### DISCUSSION TITLE
What is the "Navigation Button"?

### DISCUSSION THREAD

**Discussion by suguruwataru** [plateaukao/einkbro#275] [Q&A] [answered]
I found a setting that lets users bind some actions to swiping on the "Navigation Button". However, I can't find where the button is. I don't notice anything that's changed after turning this on. There's also a setting that changes where the "Navigation Button" locates, but I can't find a difference in the UI after changing it.

I'm using v10.11.0.

I use the Chinese version, in which it's called "导航按钮", but I searched through the code and from the localization files it seems that "Navigation Button" is what it's called in English.

**Comment by plateaukao** [comment, marked as answer]
@ii41 
If you go to settings > UI (or Behavior) and enable hide toolbar to go fullscreen, there should be a floating button to allow you back to normal screen. That's the navigation button. :)
It only appears in fullscreen mode.

**Reply by suguruwataru** [reply]
I see, thank you ^ _ ^
